packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1290,12 +1277,31 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             ...getDeviceSpec(),
         });
 
-        // logging container load stats
+        let loadSummaryNumber: number;
+        // Get the container creation metadata. For new container, we initialize these. For existing containers,
+        // get the values from the metadata blob.
+        if (existing) {
+            this.createContainerMetadata = {
+                createContainerRuntimeVersion: metadata?.createContainerRuntimeVersion,
+                createContainerTimestamp: metadata?.createContainerTimestamp,
+            };
+            // For older documents that did not have the summaryCount / summaryNumber added to their metadata, this
+            // will be initialized to 0.
+            loadSummaryNumber = metadata?.summaryNumber ?? metadata?.summaryCount ?? 0;
```

Comment:
Do we really want 0 if it's undefined? Are there a lot of undefined ones? Do we also want to maintain summaryCount for older clients for backwards compatibility?

Reply:
It is initialized to 0 for older clients for back compat. We can leave it undefined but having this will help us correlate all events for a given summary run.

Reply:
Yeah, what if an older client that just knows how to modify summaryCount summarizes, what happens? What happens when a new client decides to summarize after the old client summarizes?

New client -> old client -> new client, Will the summary number be 3 or will it be 2?

Reply:
Oh yeah, good catch! It will actually reset to 0 when old client loads because it won't find summaryCount.
back-compat! 🙄.

I will have to write both of these for a while.

Reply:
Updated it to write both. Issue to remove this once 0.59.3000 saturates - https://dev.azure.com/fluidframework/internal/_workitems/edit/244